CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log for `process_runner`
 
+## 4.2.0
+
+* Added `WorkerJobGroup` for running a group of dependent tasks in order.
```

Comment:
```suggestion
* Adds `WorkerJobGroup` for running a group of dependent tasks in order.
```

Reply:
use present tense, because this only happens when people upgrade. This is also more inline with first party package

Reply:
Done